lib/install/config/webpack/shared.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   ),
 
   output: {
-    filename: '[name].js',
+    filename: '[name]-[hash].js',
```

Comment:
This is not properly explained in doc but basically chunkhash isn't available with HMR
![screen shot 2017-08-16 at 09 56 25](https://user-images.githubusercontent.com/771039/29355694-e9435cf6-8269-11e7-8d06-3aa1fae5d512.png)